core/src/scss/elements/input/_range.scss

```diff
@@ -0,0 +1,92 @@
+@charset "UTF-8";
+
+// Range inputs
+
+[type='range'] {
+  -webkit-appearance: none;
+  border: 0;
+  padding-left: 0;
+  width: 100%;
+}
+
+[type='range']:focus {
+  box-shadow: none;
+  outline: none;
```

Comment:
Just want to double-check that this is still ok. I know this was from the previous state but I want to double-check that we haven't learned more since it was initially committed.

Reply:
Please see above.